index.ts

```diff
@@ -68,7 +78,7 @@ const run = async () => {
     repoName: repoName,
     pullRequestId: pullRequestId,
     env: env,
-    agent: 'gh-action',
+    agent: 'github',
```

Comment:
Renaming this to be consistent with [maestro](https://github.com/mobile-dev-inc/maestro/blob/main/maestro-cli/src/main/java/maestro/cli/util/CiUtils.kt#L4)

Reply:
@dmitry-zaitsev do we have any queries that rely on agent being set to `gh-action` as of today that I need to update?

Reply:
Not that I remember

Reply:
Great, thanks!